modules/go-compatibility-and-support-matrix.adoc

```diff
@@ -21,15 +21,15 @@ In {OCP} 4.13, the `stable` channel has been removed. Before upgrading to {OCP}
 
 [options="header"]
 |===
-|OpenShift GitOps 7+|Component Versions|OpenShift Versions
+|OpenShift GitOps 8+|Component Versions|OpenShift Versions
```

Comment:
One comment about the format of this table. Based on our RH guidelines: https://redhat-documentation.github.io/supplementary-style-guide/#accessibility-tables, we should avoid tables with irregular headers; and it links us to this example: https://www.w3.org/WAI/tutorials/tables/irregular/#table-with-two-tier-headers

So I think you should probably create an issue for this and fix it (but definitely not as part of this RN PR)

cc: @Srivaralakshmi